R/util.R

```diff
@@ -184,7 +184,12 @@ file_with_meta_ext <- function(file, meta_ext, ext = xfun::file_ext(file)) {
 }
 
 knitr_files_dir <- function(file) {
-  paste(xfun::sans_ext(file), "_files", sep = "")
+  if (!is.null(getOption("rmarkdown.files.suffix"))) {
+    suffix <- getOption("rmarkdown.files.suffix")
+  } else {
+    suffix <- "_files"
+  }
```

Comment:
You can just do this:

```suggestion
  suffix <- getOption("rmarkdown.files.suffix", "_files")
```